main/src/library/MutDeque.flix

```diff
@@ -309,6 +314,13 @@ namespace MutDeque {
         else
             false
 
+    ///
+    /// Returns `d` as an `Array`.
+    ///
+    pub def toArray(d: MutDeque[a]): Array[a] & Impure =
+        let MutDeque(a, f, b) = d;
+        Array.slice(deref f, deref b, deref a)
```

Comment:
Does it take too much to use the `Array.slice` on the array in `MutDeque`?
Because then I can use `Array.toIterator` for `MutDeque`

Reply:
I think that would be fine for now, however, there are some issues with `f` having a higher index than `b` when it the deque wraps around. In other words, simply calling `Array.slice` may yield the unused indices. There should be a `toList` that's safe to use.
Not sure about performance of Array.slice--I think Magnus is the authority on that :)

Reply:
A test that performs `pushFront` as the first operation and then either some `pushBack` or `pushFront` should catch this case

Reply:
That's true. I think maybe the `toList` is too expensive like for `Map`.